src/main/python/image_preprocessing/face_manager.py

```diff
@@ -2,8 +2,14 @@
 from imutils.face_utils import rect_to_bb
 from hutts_utils.hutts_logger import logger
 import dlib
+import os
 import cv2
 
+TEMPLATE_DIR = "{base_path}/../../main/python/image_preprocessing/templates/".format(
+    base_path=os.path.abspath(os.path.dirname(__file__)))
+
+face_not_found_place_holder = cv2.imread(TEMPLATE_DIR + "profile.jpg")
```

Comment:
This looks like a constant - please follow the naming conventions for constants.